src/python/pants/backend/python/tasks/pytest_run.py

```diff
@@ -260,12 +285,16 @@ def _maybe_emit_coverage_data(self, workdirs, test_targets, pex):
     pex_src_root = os.path.relpath(self._source_chroot_path, get_buildroot())
 
     src_to_target_base = {}
-    for target in test_targets:
-      libs = (tgt for tgt in target.closure()
-              if tgt.has_sources('.py') and not isinstance(tgt, PythonTests))
+    srcs_to_omit = set()
+    if self.get_options().coverage_omit_test_sources:
+      for test_target in test_targets:
+        srcs_to_omit.update(test_target.sources_relative_to_source_root())
+    for test_target in test_targets:
+      libs = (tgt for tgt in test_target.closure() if tgt.has_sources('.py'))
```

Comment:
Redudant parantheses

Reply:
They are necessary to define a comprehension. Removing them leads to syntax error.